tests/fixtures/simple/openapi.yaml

```diff
@@ -1310,6 +1310,35 @@ paths:
       responses:
         '200':
           description: Echo the validated request.
+  '/status-specific-content':
```

Comment:
Do you want to leave the single quotes off this path, to match the existing style?